### ISSUE TITLE
Add list with required files for components

### ISSUE THREAD

**Issue by JLammeer** [twbs/bootstrap#28219]
When working with `bootstrap.css` all you gotta do is, basically, copy-and-pate or use some snippet tool. There is no worry about "what components/utilities should I import in order to this component to work" appropriately.

With that in mind, as opposed to let every single developer figure it out, would be a real time saving (especially for newbies) if there were two lists, in every component documentation: **required** and **suggested** files.

As an example for `navbars`, we would have something like this:

``` sass
If you're building Bootstrap with Sass, you will need these files:

- @import "~/node_modules/bootstrap/scss/navbar";
- @import "~/node_modules/bootstrap/scss/nav";
- @import "~/node_modules/bootstrap/scss/transitions".
```

This kind of summary already exists for required JS files, as [this documentation section](https://getbootstrap.com/docs/4.2/getting-started/introduction/#js) shows. Although, it could mixed with the above list, so to avoid users needing to get back there.

```
If you're building Bootstrap with Sass, you will need these files:

- @import "~/node_modules/bootstrap/scss/navbar";
- @import "~/node_modules/bootstrap/scss/nav";
- @import "~/node_modules/bootstrap/scss/transitions".

And if you're also building JS files, these components requires:

- <script src='~/node_modules/jquery/dist/jquery.slim.js'></script>
- <script src='~/node_modules/bootstrap/js/dist/util.js'></script>
- <script src='~/node_modules/bootstrap/js/dist/collapse.js'></script>

PS: JS file imports needs to be in the given order.
```

**Comment by JLammeer** [comment]
I just noticed that, [something similar](https://getbootstrap.com/docs/4.2/components/navbar/#how-it-works) already happens, though it is not as efficient as the above idea.

With the idea stated in this issue, the " How it works" section would be way clear.

**Comment by chrisbo** [comment]
I have just gone through setting up individual JS and SCSS files for the first time and have to agree. It would save a lot of investigation and testing by improving the documentation. It is not entirely clear; what the order should be, what is required, what is optional, what depends on what, or the purpose of each file. I envisage a cheat sheet of sorts could fully cover this.

**Comment by bardiharborow** [comment]
Closed as a duplicate of #23813.
